docs/3.0.0-beta.x/getting-started/install-requirements.md

```diff
@@ -14,11 +14,9 @@ You can install `Node.js` and `npm` for `Windows 10`, `Ubuntu 18.04` and `Mac O/
 
 #### Installation Instructions for each Operating System:
 
-:::: tabs cache-lifetime="10" :options="{ useUrlFragment: false }"
+:::: tabs
 
-::: tab "Windows 10" id="windows-requirements"
-
-**WINDOWS 10**
+::: tab "Windows 10"
```

Comment:
Sometimes we use WIndows other times windows 10. We should be consistent everywhere